pkg/controllers/crds/crds_controller.go

```diff
@@ -227,7 +230,12 @@ func (r *Reconciler) updateCRD(ctx context.Context, req ctrl.Request) error {
 			return err
 		}
 	}
-	return r.Update(ctx, &updatedResource)
+	if !equality.Semantic.DeepEqual(before, &updatedResource) {
+		defer log.Info("updated crd")
```

Comment:
why defer log here? 🤔  Just log it?

Reply:
err yeah got too cute and wanted it to log after the Update call, but we don't really want it if the error happens.